src/java.desktop/unix/native/libawt_xawt/xawt/XToolkit.c

```diff
@@ -692,7 +692,9 @@ void awt_output_flush() {
 static void wakeUp() {
     static char wakeUp_char = 'p';
     if (!isMainThread() && awt_pipe_inited) {
-        write ( AWT_WRITEPIPE, &wakeUp_char, 1 );
+        if (write ( AWT_WRITEPIPE, &wakeUp_char, 1 ) != 1) {
+            fprintf(stderr, "Cannot not write to AWT utility control: %s\n", strerror(errno));
```

Comment:
what happens to the process if this call fails ?
Can it continue ?
You can check by commenting out the call and see what happens when you run AWT tests
And assuming it is a continuable error, can you do the print only in a DEBUG build ?